Tests/Unit/EmogrifierTest.php

```diff
@@ -1519,4 +1519,45 @@ public function emogrifyAutomaticallyClosesUnclosedTag()
             $this->subject->emogrify()
         );
     }
+
+    /**
+     * @test
+     */
+    public function emogrifyDefaultReturn()
```

Comment:
Generally, test names should be complete sentences (though maybe a git Pidgin-English style).

Better name:

emogrifyReturnsCompleteHtmlDocument